src/main/java/nl/tudelft/ewi/devhub/server/database/entities/Course.java

```diff
@@ -70,6 +70,10 @@
 	@OneToMany(mappedBy = "course", fetch = FetchType.LAZY)
 	private List<CourseAssistant> courseAssistants;
 
+	public void setCode(String code) {
+		this.code = code.toUpperCase();
```

Comment:
Now enforcing unique course code (**case insensitive**, as the repository names are care insensitive).